diff --git a/node_modules/@types/discord-rpc/index.d.ts b/node_modules/@types/discord-rpc/index.d.ts
index 0ea8e3b..266779c 100644
--- a/node_modules/@types/discord-rpc/index.d.ts
+++ b/node_modules/@types/discord-rpc/index.d.ts
@@ -8,7 +8,8 @@
 //                 HanchaiN <https://github.com/HanchaiN>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { EventEmitter, Listener } from 'events';
+import { EventEmitter } from 'events';
+type Listener = (...args: any[]) => void;
 
 export function register(id: string): boolean;
 
